doc-api/source/utilization.rst

```diff
@@ -0,0 +1,48 @@
+Utilization
+===========
+
+2 ways to use `PBxplore`:
+  - command-line scripts
+  - API
+
+Command-line scripts
+--------------------
+
+PBxplore is bundle with a suite of command-line tools.
+Once installed, they should be available in your ``$PATH``.
+Here the list:
+
+- :doc:`PBassign <PBassign>` assign :doc:`Protein Blocks (PBs) <intro_PB>`
+  from a set of protein structures.
+- :doc:`PBcount <PBcount>` computes the frequency of PBs at each position
+  along the amino acid sequence.
+- :doc:`PBstat <PBstat>` generates frequency and logo plots, and estimates the :ref:`Neq <Neq>`.
+- :doc:`PBclust <PBclust>` use clustering algorithm (k-means) to re-group similar PBs sequences.
+
+
+API
+---
+
```

Comment:
Oh yeah you're right. I thought it was the `index.rst`.
Yes, there is nothing yet. We need to write a paragraph about the API (it's in the TODO list) and yes, we could add at the end links to notebook